Staccato-iOS/Staccato-iOS/Util/Model/Image/ImageResponse.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  ImageResponse.swift
+//  Staccato-iOS
+//
+//  Created by 강재혁 on 1/9/25.
+//
+
+import Foundation
+
+struct ImageURL: Codable {
+    let ImageURL: String            // 이미지 URL
```

Comment:
LowerCamelcase를 지켜서
```suggestion
    let imageURL: String            // 이미지 URL
```
로 사용하는 것은 어떨까요?

Reply:
Response 혹은 Request가 저 형태로 되어야 한다면, CodingKey를 사용해야 할 것 같습니다!

Reply:
넵 오타인것 같습니다. 아마 변수명을 imageUrl로 맞추겠습니다.